inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -126,9 +131,27 @@ def name2marc(self, key, value):
 @utils.for_each_value
 def ids(self, key, value):
     """All identifiers, both internal and external."""
+    IDS_MAP = {
+        'ARXIV': 'ARXIV',
+        'BAI': 'INSPIRE BAI',
+        'CERN': 'CERN',
+        'DESY': 'DESY',
+        'INSPIRE': 'INSPIRE ID',
+        'KAKEN': 'KAKEN',
+        'ORCID': 'ORCID',
+        'SLAC': 'SLAC',
+        'WIKIPEDIA': 'WIKIPEDIA',
+    }
+
+    type_ = IDS_MAP.get(value.get('9').upper())
```

Comment:
This is risky. See: https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/602792/